doc/chips/10.rst

```diff
@@ -250,6 +250,78 @@ block while the rules for Phase 2 may be applied to the second, without a
 constant check to the location of the ``.init()`` call which is the linchpin for
 syntax 1.
 
+Child's fields could depend on parent's fields
```

Comment:
Organizationally, I would prefer the titles of sections to be more concise.  This section is intended to be related to the previous (syntax) section, correct?  But also to the decision about when a child's fields are initialized?  The title makes it feel like a completely different direction from the syntax section, but you tie it more strongly together in the body.  I'd prefer it closer to the description of field dependence, but since you rely on our syntax description, perhaps this would be better suited as part of a new section at the end of the document entitled "Alternate Decisions", which could then be divided into "Parent/Child Field Dependence" and "Ordering of Parent/Child Initialization"?